src/main/java/org/mule/module/xml/internal/operation/XPathOperation.java

```diff
@@ -79,9 +79,11 @@ public XPathOperation(EntityExpansion expandEntities) {
                                    @Optional @Content @NullSafe Map<String, Object> contextProperties,
                                    @Optional @NullSafe List<NamespaceMapping> namespaces,
                                    @Optional @ParameterDsl(allowInlineDefinition = false) NamespaceDirectory namespaceDirectory,
-                                   @Config XmlModule config) {
+                                   @Config XmlModule config,
+                                   @Optional(defaultValue = "false") boolean keepTrailingNewlines) {
```

Comment:
need to add javadoc for this param explaining what it does. Also the name is very confusing. Shouldn't it be something more like `add trailing new lines` or something like that?